src/commands/profile/current.ts

```diff
@@ -0,0 +1,38 @@
+import {command, metadata, option} from 'clime';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'd',
+        description: 'Default profile name',
+    })
+    profile: string;
+}
+
+@command({
+    description: 'Default profile name',
+})
+export default class extends ProfileCommand {
+    constructor() {
+        super();
+    }
+
+    @metadata
+    execute(options: CommandOptions) {
+        const profile = this.getCurProfile();
+
+        if (undefined === options.profile || '' === options.profile) {
+            console.log(`default profile name: ${profile.name}`);
```

Comment:
try to use ' instead of `

Reply:
I'm not very clear how to use `instead of`

Reply:
Instead of using a grave accent or back quote (`) to wrap the string, try to use single quote symbol (')